underscore.js

```diff
@@ -96,21 +96,26 @@
 
   // Similar to ES6's rest params (http://ariya.ofilabs.com/2013/03/es6-and-rest-parameter.html)
   // This accumulates the arguments passed into an array, after a given index.
-  _.restParams = function rest(func, startIndex) {
-    startIndex = startIndex != null ? +startIndex : 1;
+  _.restParams = function(func) {
+    var startIndex = func.length - 1;
```

Comment:
What sense would it be to have it be anything but the last arg? The whole method would have to be rewritten to support it any place else.